packages/react-dogfood/hooks/useKeyboardShortcuts.ts

```diff
@@ -0,0 +1,83 @@
+import {
+  defaultReactions,
+  useCall,
+  useToggleAudioMuteState,
+  useToggleVideoMuteState,
+} from '@stream-io/video-react-sdk';
+import { useEffect } from 'react';
+
+import hotkeys from 'hotkeys-js';
+
+enum Hotkeys {
+  TOGGLE_AUDIO_MAC = 'cmd+d',
```

Comment:
Can we also have "cmd+space"? I think that is a more common shortcut that would be easier to remember.

Reply:
I used the ones used in the [Google Meet](https://support.google.com/a/users/answer/9896256?hl=en) - would you rather me to switch to the Slack preset?

Reply:
can we have both? 😅

Reply:
![image](https://media.tenor.com/odyVsZbC-OYAAAAC/why-not-both-why-not.gif)